docs/src/main/asciidoc/grpc-service-implementation.adoc

```diff
@@ -521,29 +521,47 @@ quarkus.http.auth.basic=true <1>
 ----
 package org.acme.grpc.auth;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
 
+import com.rajant.connector.proto.Greeter;
```

Comment:
Should we use a more neutral package name? Like org.acme?

Reply:
Oops yes sorry that should be org.acme
I’ll fix it in a few mins

Reply:
Fixed